protocols/imap/src/test/java/org/apache/james/imap/processor/ListRightsProcessorTest.java

```diff
@@ -98,7 +100,10 @@ public void setUp() throws Exception {
         listRightsRequest = new ListRightsRequest(TAG, MAILBOX_NAME, USER_1.asString());
 
         user1Key = EntryKey.deserialize(USER_1.asString());
-        listRights = new Rfc4314Rights[] {Rfc4314Rights.fromSerializedRfc4314Rights("ae"), Rfc4314Rights.fromSerializedRfc4314Rights("i"), Rfc4314Rights.fromSerializedRfc4314Rights("k")};
+        listRights = List.of(
```

Comment:
ImmutableList?

Reply:
in a Test class? Why would it matter?

Reply:
Because I more have the habbit of it and know what to expect when I see it, but that's ok.

Reply:
For sure, java 11 adoption is less than week old (:

Reply:
Ah that's specific to Java 11 ;-)